packages/commons/src/config/selfcareConfig.ts

```diff
@@ -10,4 +10,16 @@ export const SelfCareConfig = z
     selfcareApiKey: c.SELFCARE_V2_API_KEY,
   }));
 
-export type SelfCareConfig = z.infer<typeof SelfCareConfig>;
+export type SelfCareClientConfig = z.infer<typeof SelfCareClientConfig>;
+
+export const SelfcareConsumerConfig = z
+  .object({
+    SELFCARE_TOPIC: z.string(),
+    INTEROP_PRODUCT: z.string(),
+  })
+  .transform((c) => ({
+    selfcareUsersTopic: c.SELFCARE_TOPIC,
```

Comment:
```suggestion
    selfcareTopic: c.SELFCARE_TOPIC,
```
Since this is the `commons` configuration it's best to keep it generic